frontend/src/pages/detail/BBDetail.tsx

```diff
@@ -188,8 +192,38 @@ export default function BBDetail() {
   const deleteSale = (rowData: BBSaleRow) => {
     const _data = sales.filter((val) => val.id !== rowData.id);
     setSales(_data);
+    setTotalRevenue(calculateTotal(_data));
   };
 
+  // Called to make delete pop up show
+  const deleteBuyBackPopup = () => {
+    logger.debug("Delete Sales Reconciliation Clicked");
+    setDeletePopupVisible(true);
+  };
+
+  // Call to actually delete the element
+  const deleteBuyBackFinal = () => {
+    logger.debug("Delete Buy Back Finalized");
+    setDeletePopupVisible(false);
+    BUYBACK_API.deleteBuyBack({
+      id: id!,
+    })
+      .then(() => {
+        showSuccess(toast, "Buy Back Sale Deleted");
```

Comment:
I think we can change this to "Book Buyback Deleted/Book Buyback failed to delete". This goes for all instances of Buyback, he has it as one word in the requirements and refers to it as a "Book Buyback" so I think we should do the same thing.